src/components/FavoritesLink/index.js

```diff
@@ -1,25 +1,23 @@
-import { useState } from 'react';
-import { FavoritesLink, FavoritesSidebarContainer } from "./styled";
+import { FavoritesList, FavoritesSidebarContainer } from "./styled";
 import useAuth from '../../hooks/useAuth';
-import Sidebar from "../FavoritesSidebar";
+import SidebarPanel from "../FavoritesSidebar";
 import useUserLikes from "../../hooks/useUserLikes";
+import { useContext } from "react";
+import { GlobalContext } from "../../context/GlobalContext";
 
-export default function Favorites() {
-    const [showSidebar, setShowSidebar] = useState(false);
+export default function FavoritesPanel() {
     const { user } = useAuth();
     const { initialLikes } = useUserLikes(user);
-
+    const { isSidebarOpen } = useContext(GlobalContext);
     return !user || !initialLikes ? null : (
         <>
-            <FavoritesLink onClick={() => setShowSidebar(!showSidebar)}>
+            <FavoritesList>
```

Comment:
is it a list or a title? Looks like the result will be:

```
<ul>Favoritos</ul>
```

Reply:
my bad, i forgot to remove from the code